app/Controller/DirectoryController.php

```diff
@@ -121,20 +121,32 @@ public function branch(Request $request, Response $response, array $arguments)
         $repositoryFinder = new RepositoryFinder($gitWrapper);
         $gitRepository = $repositoryFinder->getGitRepositories($absolutePath, $settings['virtual-host'])->getIterator()->current();
         $branches = $gitRepository->branch(['r']);
+        $tags = $gitRepository->tag(['l', 'sort=-v:refname']);
+        $currentBranch = $gitRepository->getCurrentBranch();
+
+        // remove heads/ prefix if it was an tag
+        $currentBranch = str_replace('heads/', '', $currentBranch);
 
         // Remove all HEAD pointers
         $branches = array_filter($branches, function ($value) {
             return strpos($value, '/HEAD ') === false;
         });
 
+        // add tags string on every tag
+        array_walk($tags, function (&$value, $key) {
+            $value = 'tags/' . $value;
```

Comment:
I still would prefer to prefix branches instead of tags. Is there any reason that you prefer prefixing the tags?